tests/test_visitors/test_ast/test_subscripts/test_undescriptive_slicer_operation.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    UndescriptiveSliceOperationViolation,
+)
+from wemake_python_styleguide.visitors.ast.subscripts import SubscriptVisitor
+
+usage_template = 'constant[{0}]'
+
+
+@pytest.mark.parametrize('expression', [
+    '::-1',
+    ':',
+    '::',
+])
+def test_undescriptive_slice_operation(
+    assert_errors,
+    parse_ast_tree,
+    expression,
+    default_options,
+):
+    """Testing that undescriptive slice operations are forbidden."""
+    tree = parse_ast_tree(usage_template.format(expression))
+
+    visitor = SubscriptVisitor(default_options, tree=tree)
+    visitor.run()
+
+    assert_errors(visitor, [UndescriptiveSliceOperationViolation])
+
+
+@pytest.mark.parametrize('expression', [
+    '1:2',
+    '::3',
+    '4',
+    '3:7',
+    '3:7:2',
+    '3:',
+    ':7',
+    '3::2',
+    '5:7:',
+    ':7:2',
+    '3::',
```

Comment:
Looks like this subscript also does not make any sense. Why would anyone want to write `3::` and not just `3:`?

Reply:
Related https://github.com/wemake-services/wemake-python-styleguide/issues/1071